website/go.mod

```diff
@@ -10,9 +10,13 @@ require (
 )
 
 require (
+	github.com/google/docsy v0.2.0 // indirect
```

Comment:
What will prevent `go mod tidy` from removing this dependency? It's not used by anything that Go can see.

Reply:
Good point. This is an issue indeed, please note that there is a related open [issue](https://github.com/gohugoio/hugo/issues/6115) in the hugo project that addresses this topic.
Since go 1.14, there is a [-modfile](https://tip.golang.org/doc/go1.14#go-flags) flag that instructs the go command to use an alternate `go.mod` file. Could this be a solution?